src/compiler.rs

```diff
@@ -1367,11 +1373,13 @@ fn generate_entry_point_wrapper<'c>(
             ),
             (
                 Identifier::new(context, "llvm.linkage"),
-                Attribute::parse(context, "#llvm.linkage<private>").unwrap(),
+                Attribute::parse(context, "#llvm.linkage<private>")
+                    .expect("hardcoded attribute is valid"),
             ),
             (
                 Identifier::new(context, "llvm.CConv"),
-                Attribute::parse(context, "#llvm.cconv<fastcc>").unwrap(),
+                Attribute::parse(context, "#llvm.cconv<fastcc>")
+                    .expect("hardcoded attribute is valid"),
```

Comment:
shouldn't it be "invalid" instead of "valid"?

Reply:
You are right. FIxed

Reply:
I changed the code to use that error variant. Thanks for pointing it out.